Supplementary resources Unit 4: Multimodal Models (#173)
johko/computer-vision-course: chapters/en/Unit 4 - Mulitmodal Models/supplementary-material.mdx

> # Supplemental reading and resources 🤗

Reviewer: ```suggestion
# Supplementary Reading and Resources 🤗
```

> We hope that you found the unit on Multimodal Models exciting. If you'd like to learn and explore in detail about Multimodal Learning and Models, here is a list of resources

Reviewer: ```suggestion
We hope that you found the unit on multimodal models exciting. If you'd like to learn and explore in detail about multimodal learning and models, here is a list of resources
```

for your reference:

> 1. [**Hugging Face Tasks**](https://huggingface.co/tasks) offers an overview of a wide variety of tasks related to domains like Computer Vison, Audio, NLP, Multimodal Learning and Reinforcement Learning. The tasks contain demos, use-cases, models, datasets etc.

Reviewer: ```suggestion
- [**Hugging Face Tasks**](https://huggingface.co/tasks) offers an overview of various tasks under domains like Computer Vision, Audio, NLP, Multimodal Learning and Reinforcement Learning. The tasks contain demos, use cases, models, datasets, etc.
```

> 2. [**11-777 MMML**](https://cmu-multicomp-lab.github.io/mmml-course/fall2022/) course on Multimodal Machine Learning by CMU. You can find the video lectures [**here**](https://www.youtube.com/@LPMorency/playlists).

Reviewer: ```suggestion
- [**11-777 MMML**](https://cmu-multicomp-lab.github.io/mmml-course/fall2022/) course on multimodal machine learning by CMU. You can find the video lectures [**here**](https://www.youtube.com/@LPMorency/playlists).
```

> 3. [**Blog on Multimodality and LLMs by Chip Huyen**](https://huyenchip.com/2023/10/10/multimodal.html) provides a comprehensive overview of multimodality, large multimodal models, systems like BLIP, CLIP, etc.

Reviewer: ```suggestion
- [**Blog on Multimodality and LLMs by Chip Huyen**](https://huyenchip.com/2023/10/10/multimodal.html) provides a comprehensive overview of multimodality, large multimodal models, systems like BLIP, CLIP, etc.
```

> 4. [**Awesome Multimodal ML**](https://github.com/pliang279/awesome-multimodal-ml), a GitHub repository containing papers, courses, architectures, workshops, tutorials etc.

Reviewer: ```suggestion
- [**Awesome Multimodal ML**](https://github.com/pliang279/awesome-multimodal-ml) is a GitHub repository containing papers, courses, architectures, workshops, tutorials, etc.
```

> 5. [**Awesome Multimodal Large Language Models**](https://github.com/BradyFU/Awesome-Multimodal-Large-Language-Models), a GitHub repository containing papers and datasets related to multimodal LLMs. 

Reviewer: ```suggestion
- [**Awesome Multimodal Large Language Models**](https://github.com/BradyFU/Awesome-Multimodal-Large-Language-Models), a GitHub repository containing papers and datasets related to multimodal LLMs. 
```

> 6. [**EE/CS 148, Caltech**](https://gkioxari.github.io/teaching/cs148/) course on Large Language and Vision Models.

Reviewer: ```suggestion
- [**EE/CS 148, Caltech**](https://gkioxari.github.io/teaching/cs148/) course on Large Language and Vision Models.
```

# Supplemental reading and resources 🤗

We hope that you found the unit on Multimodal Models exciting. If you'd like to learn and explore in detail about Multimodal Learning and Models, here is a list of resources
for your reference:

> 1. [**Hugging Face Tasks**](https://huggingface.co/tasks) hosts a wide variety of tasks related to domains like Computer Vison, Audio, NLP, Multimodal Learning and Reinforcement Learning. The tasks contain demos, use-cases, models, datasets etc.

Reviewer 2: I wouldn't say it "hosts" a variety of tasks. Rather, the page offers an overview of a wide variety of tasks.

Author: Thank you @MKhalusova I changed it 😄

2. [**11-777 MMML**](https://cmu-multicomp-lab.github.io/mmml-course/fall2022/) course on Multimodal Machine Learning by CMU. You can find the video lectures [**here**](https://www.youtube.com/@LPMorency/playlists).

> 3. [**Blog on Multimodality and LLMs by Chip Huyen**](https://huyenchip.com/2023/10/10/multimodal.html) provides an comprehensive overview of multimodality, large multimodal models, systems like BLIP, CLIP etc.

Reviewer 2: ```suggestion
3. [**Blog on Multimodality and LLMs by Chip Huyen**](https://huyenchip.com/2023/10/10/multimodal.html) provides a comprehensive overview of multimodality, large multimodal models, systems like BLIP, CLIP, etc.
```

Author: Done 😄

4. [**Awesome Multimodal ML**](https://github.com/pliang279/awesome-multimodal-ml), a GitHub repository containing papers, courses, architectures, workshops, tutorials etc.
5. [**Awesome Multimodal Large Language Models**](https://github.com/BradyFU/Awesome-Multimodal-Large-Language-Models), a GitHub repository containing papers and datasets related to multimodal LLMs. 
6. [**EE/CS 148, Caltech**](https://gkioxari.github.io/teaching/cs148/) course on Large Language and Vision Models.

> 7. [**16-892 CMU**](https://www.cs.cmu.edu/~deva/teach/16-892/index.html) seminar on Multimodal Foundation Models, Fall 2023. 

Reviewer 2: The sign-up link to this course is no longer active, and the "Lectures" link leads to a spreadsheet with the course structure but no actual lectures. This doesn't seem useful.

Author: Yes @MKhalusova it makes sense, I will remove this course. It had a reading list as well but since the course is inactive I will just remove it.
